modules/base-realms/realm-otp/realm.tf

```diff
@@ -0,0 +1,41 @@
+module "realm" {
+  source       = "../../realm"
+  realm_name   = var.realm_name
+  display_name = "Email"
+}
+
+data "keycloak_authentication_execution" "browser_identity_provider_redirector" {
+  realm_id          = module.realm.id
+  parent_flow_alias = "browser"
+  provider_id       = "identity-provider-redirector"
+}
+
+resource "keycloak_authentication_execution_config" "browser_identity_provider_redirector_config" {
+  realm_id     = module.realm.id
+  execution_id = data.keycloak_authentication_execution.browser_identity_provider_redirector.id
+  alias        = var.realm_name
+  config = {
+    defaultProvider = var.realm_name
+  }
+}
+
+resource "keycloak_openid_client" "logout_redirect_uri_delegator" {
+  realm_id = module.realm.id
+
+  client_id   = "logout-redirect-uri-delegator"
+  name        = "logout redirect uri delegator"
+  description = "this client allows post-logout redirect URIs within the KC server"
+
+  enabled                      = false
+  standard_flow_enabled        = true
+  implicit_flow_enabled        = false
+  direct_access_grants_enabled = false
+  service_accounts_enabled     = false
+
+  access_type = "CONFIDENTIAL"
```

Comment:
This access type is confidential because it's connecting otp app and otp realm correct?

Reply:
This client is used during logout phase and each of the oidc integrations in parent realms that we have use this client